File: python/twosum.py
class Solution(object):
    def twoSum(self, nums, target):
        hashmap={}
        for i in range(len(nums)):
            if target-nums[i] in hashmap:
                return i,hashmap[target-nums[i]]
            else:
                hashmap[nums[i]]=i
                

solution = Solution()
nums = [-3,4,3,90]

indices = solution.twoSum(nums, 0)
if indices is None:
    print("No match found")
else:
    print("The indices are:", indices)

File: python/1-two-sum/two-sum.java
import java.util.HashMap;
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> harsh= new HashMap<>();
        for(int i=0;i<nums.length;i++){
            if(harsh.containsKey(target-nums[i])){
                int[] arr=new int[]{i,harsh.get(target-nums[i])};
                return arr;
            }
            harsh.put(nums[i],i);
        }
        return new int[]{-1,-1};
    }
}